.gitattributes

```
@@ -0,0 +1,6 @@
+*	text=auto eol=lf
+*.md	linguist-detectable
+CODE_OF_CONDUCT.md	-linguist-detectable
+LICENSE.md	-linguist-detectable
+README.md	-linguist-detectable
+PEERS_GUIDELINES.md	-linguist-detectable
```

My suggestion would be to add only `files/**`, regardless of their type:

```suggestion
files/** linguist-detectable
```

This prevents `docs/` from being counted, and reveals JSON, Text and SVG files in the `files/` folder.

---

@SphinxKnight thoughts on the above?

---

My initial intention was to follow what's set up for [mdn/content](https://github.com/mdn/content/blob/main/.gitattributes) so that any later change there can be applied here (if relevant).
Regarding the `docs` exclusion, I understood from the linguist documentation that it was implicitly excluded. I've nothing changing this "implicit" for an explicit exclusion.

---

@caugner when you have a second.